inc/az_ulib_ipc_api.h

```diff
@@ -77,7 +77,7 @@ static inline az_ulib_result az_ulib_ipc_init(az_ulib_ipc* ipc_handle) {
  *
  * 1) Stop all threads that make calls to the published interfaces.
  * 2) Finalize or cancel all asynchronous calls, and ensure that their callbacks were called.
- * 3) Unsubscribe all events.
+ * 3) Unsubscribe all telemetries.
```

Comment:
should this be telemetries? maybe capabilities?

Reply:
Unsubscribe is only for telemetry, not for the other capabilities. 

---
In reply to: [561177490](https://github.com/Azure/azure-ulib-c/pull/55#discussion_r561177490) [](ancestors = 561177490)